webapp/src/pages/GameRulesPage/GameRulesPage.tsx

```diff
@@ -0,0 +1,148 @@
+          <Text variant={TextVariant.SUBHEADING}>Gameplay:</Text>
+          <ol>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                At the start of the game, both players need to click the "Ready"
+                button to indicate they are ready to play.
+              </Text>
+            </li>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                Once both players have clicked the "Ready" button, the game
+                starts.
+              </Text>
+            </li>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                The players move their paddles using the left and right arrow
+                keys on their keyboard, or by touching and dragging the paddle
+                on a mobile device.
+              </Text>
+            </li>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                The ball bounces off the paddles and the walls of the game
+                screen.
+              </Text>
+            </li>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                Each time a player misses the ball, the other player scores a
+                point.
+              </Text>
+            </li>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                The game continues until one player reaches 10 points and wins
+                the game.
+              </Text>
+            </li>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                If one of the players disconnects during the game, the game will
+                be paused for up to 30 seconds or until the disconnected player
+                reconnects.
+              </Text>
+            </li>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                If the disconnected player reconnects within the 30 seconds, the
+                game will resume from where it was paused.
+              </Text>
+            </li>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                If the disconnected player does not reconnect within the 30
+                seconds, the game will end and the player who did not disconnect
+                will be declared the winner.
+              </Text>
+            </li>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                If a player chooses to quit the game at any point, the other
+                player will be declared the winner.
+              </Text>
+            </li>
+          </ol>
+        </div>
+        <div className="game-rules-page-main-scoring">
+          <Text variant={TextVariant.SUBHEADING}>Scoring:</Text>
+          <ol>
+            <li>
+              <Text variant={TextVariant.PARAGRAPH}>
+                A player scores a point when the ball passes their opponent's
+                paddle.
```

Comment:
This was more or less said in the 5th item above

Reply:
Yes, but this is a different section. If we leave the section, we can update it when the explanations will diverge. If we don't, then it will stay messy.